lib/remote/configpackageutility.cpp

```diff
@@ -198,7 +210,8 @@ void ConfigPackageUtility::TryActivateStageCallback(const ProcessResult& pr, con
 	}
 }
 
-void ConfigPackageUtility::AsyncTryActivateStage(const String& packageName, const String& stageName, bool activate, bool reload)
+void ConfigPackageUtility::AsyncTryActivateStage(const String& packageName, const String& stageName, bool activate, bool reload,
+	Shared<Defer>::Ptr resetPackageUpdates)
```

Comment:
Can (and therefore should) be a const ref. Btw.: explain in your own words why.

Reply:
Well 🫤, since the deferred callback isn't going out of scope prior to the `AsyncTryActivateStage()` starting a new process, so it's fine to make it const ref, I guess.

Reply:
`const Ptr&` is probably more consistent with how it's used in other places.